### PR TITLE
rewrite the verify and integration job as a good prow citizen

### PR DESCRIPTION
**Pull request by krzyzacy**
(we can ditch the scenario once this works)

/assign @BenTheElder @ixdy @cblecker

### CODE REVIEW

**Comment by ixdy** [comment]
I think the integration jobs also use the verify scenario (for reasons) so we'll need to fix those too?

**Comment by krzyzacy** [comment]
good point

**Comment by krzyzacy** [comment]
also added a canary integration job, also looks like the -prow variant is not used anymore, replaced them with -podutil editions :-)

File: config/jobs/kubernetes/sig-testing/verify.yaml [hunk 1]
```diff
@@ -30,26 +30,31 @@ presubmits:
           requests:
             cpu: 4
 
-  - name: pull-kubernetes-verify-prow
+  # TODO(krzyzacy): add branches once this works
+  # TODO(krzyzacy): Consider combine kubekins-e2e and kubekins-test
+  - name: pull-kubernetes-verify-podutil
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-testing/verify.yaml]: happy to add this, but IIRC it depends on golint being in the image so

Comment (krzyzacy) [reply config/jobs/kubernetes/sig-testing/verify.yaml]: it's currently using kubekins-test which has golint :-)

**Pull request merged**